constants/faculty_designations.py

```diff
@@ -56,4 +57,12 @@
     (VISITING_PROFESSOR, 'Visiting professor'),
     (VISITING_SCIENTIST, 'Visiting scientist'),
     (WORKSHOP_SUPERINTENDENT, 'Workshop superintendent'),
+    (PROFESSOR_OF_PRACTICE,'Professor of practice'),
 )
+
+ACTIVE_FACULTY=[ASSISTANT_PROFESSOR,ASSISTANT_PROFESSOR_CONTRACT,ASSISTANT_WORKSHOP_SUPERINTENDENT,ASSOCIATE_PROFESSOR,
+                CHAIR_PROFESSOR,CHIEF_SCIENTIFIC_OFFICER,DST_INSPIRE_FACULTY,GUEST_FACULTY,HEAD_OF_CENTRE,HEAD_OF_DEPARTMENT,JOINT_PROFESSOR,
+                LECTURER,PROFESSOR,RAMANUJAN_FELLOW,RAMALINGASWAMI_FELLOW,SCIENTIFIC_OFFICER,SENIOR_SCIENTIFIC_OFFICER,SYSTEM_PROGRAMMER,
+                VISITING_PROFESSOR,WORKSHOP_SUPERINTENDENT]
+
+VISTING_FACULTY=[VISITING_PROFESSOR,VISITING_SCIENTIST]
```

Comment:
add a new line

Reply:
Also please mention where you are using the new constans `ACTIVE_FACULTY` and `VISITING_FACULTY`

Reply:
fix typo in `VISTING_FACULTY` to `VISITING_FACULTY`